tensorrt_llm/bench/dataset/prepare_real_data.py

```
@@ -0,0 +1,314 @@
+                *dataset_config.query,
+                split=dataset_config.split,
+                streaming=True,
+                trust_remote_code=True,
+            )
+        )
+    except ValueError as e:
+        if "Config" in e:
+            e += "\n Please add the config name to the dataset config yaml."
+        elif "split" in e:
+            e += "\n Please specify supported split in the dataset config yaml."
+        raise ValueError(e)
```

Ruff (0.14.5)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)